<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-E83PQSV" failures="1" tests="3" name="Test" time="9.302" errors="0" timestamp="2024-11-19T00:34:31 IST">
  <testcase classname="TestNg.OrangeHRMListnerPgm" name="testLogo" time="0.062"/>
  <testcase classname="TestNg.OrangeHRMListnerPgm" name="testAppURL" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at TestNg.OrangeHRMListnerPgm.testAppURL(OrangeHRMListnerPgm.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppURL -->
  <testcase classname="TestNg.OrangeHRMListnerPgm" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
